File: client/src/app/models/car.ts
import { CarImage } from './carImage';
export interface Car {
  id: number;
  modelYear: number;
  brandName: string;
  colorName: string;
  dailyPrice: number;
  findexPuan: number;
  description: string;
  brandId: number;
  images: CarImage[];
  colorId: number;
  isRentable: boolean;
}

File: Entities/Concrete/Car.cs
using System;
using System.Collections.Generic;
using System.Text;
using Core.Entities;

namespace Entities.Concrete
{
    public class Car: IEntity
    {
        public int Id { get; set; }
        public int BrandId { get; set; }
        public int ColorId { get; set; }
        public int ModelYear { get; set; }
        // Boş ise database üzerinde default olarak 1000 
        public int? FindexPuan { get; set; }
        public decimal DailyPrice { get; set; }
        public string Description { get; set; }
        
    }
    
    
}
